src/main/java/net/minecraftforge/fml/client/event/RegisterReloadListenersEvent.java

```diff
@@ -0,0 +1,27 @@
+package net.minecraftforge.fml.client.event;
+
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraft.resources.IFutureReloadListener;
+import net.minecraft.resources.IReloadableResourceManager;
+
+/**
+ * Fired just before {@link IReloadableResourceManager#initialReload} so mods can register listeners.
+ * Fired on the mod event bus.
+ *
+ * Called on {@link Dist#CLIENT} - the game client.
```

Comment:
What about the server? Does it not have an initial reload?

Reply:
This event is fired from `Minecraft`, so no.

It would be useful to have a similar (or the same) event fired on the server though.